### PR TITLE
Technopig/crippling shout

### PR DESCRIPTION
**Pull request by Technopig1992**

### CODE REVIEW

File: mapdata/WarcraftLegacies/UnitData/Skin/1329803368.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Frostwolf Animal Tammer.|n|cfff5962dAbilities:|R Charge, War Cadence|n |n|cff99b4d1Can only train 2.|n|cffffcc00Attacks land and air units.|r"
+      "Value": "Frostwolf Animal Tammer.|n|cfff5962dAbilities:|R Crippling Shout, War Cadence|n |n|cff99b4d1Can only train 2.|n|cffffcc00Attacks land and air units.|r"
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/UnitData/Skin/1329803368.json:46]: Could we fix the typos while we're at it, apparently I missed them:
```suggestion
      "Value": "Frostwolf animal tamer.|n|cfff5962dAbilities:|R Crippling Shout, War Cadence|n |n|cff99b4d1Can only train 2.|n|cffffcc00Attacks land and air units.|r"
```

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Everything looks really solid aside from the fact that it applies Howl of Terror on top of Cripple. Please test that your Targets Allowed change worked then let me know.

**Comment by Technopig1992** [comment]
> Everything looks really solid aside from the fact that it applies Howl of Terror on top of Cripple. Please test that your Targets Allowed change worked then let me know.

Tested, will start making it a part of the practice going forward.  all working okay. its removing the AOE on the ability that avoids it

**Comment by YakaryBovine** [comment]
Well done. Not quite there yet. With a range of 0, the ability can still target the caster. So:
image
I would recommend changing Targets Allowed to Bridge or something.

**Comment by Technopig1992** [comment]
hmm didn't notice that ill make it cripple and it should be fine

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
The last change has clearly not been tested and this is not ready for review.

**Review by YakaryBovine — APPROVED** [review]
Looking good.

**Pull request merged**